dist-material/release-docs/LICENSE

```diff
@@ -304,12 +306,13 @@ The text of each license is the standard Apache 2.0 license.
     HikariCP 3.1.0: https://github.com/brettwooldridge/HikariCP, Apache 2.0
     zipkin 2.9.1: https://github.com/openzipkin/zipkin, Apache 2.0
     sharding-jdbc-core 2.0.3: https://github.com/sharding-sphere/sharding-sphere, Apache 2.0
-    kubernetes-client 4.0.0: https://github.com/kubernetes-client/java, Apache 2.0
+    kubernetes-client 8.0.0: https://github.com/kubernetes-client/java, Apache 2.0
     proto files from istio/istio: https://github.com/istio/istio  Apache 2.0
     proto files from istio/api: https://github.com/istio/api      Apache 2.0
     consul-client 1.2.6: https://github.com/rickfast/consul-client, Apache 2.0
     okhttp 3.9.0: https://github.com/square/okhttp, Apache 2.0
     prometheus client_java 0.6.0: https://github.com/prometheus/client_java, Apache 2.0
+    nacos 1.2.0: https://github.com/alibaba/nacos, Apache 2.0
```

Comment:
Nacos should be removed too.

Reply:
fix it,I already know how to add a license, but how to check whether the license needs to be deleted, is there a shell script to check it?

Reply:
I am afraid there is. The way is acorss-check the LICENSE file and known-oap-backend-dependencies.txt files. They should match.

Reply:
which shell script? Is it in the repository? this will help us to check the dependencies before commit